README.md

```diff
@@ -33,6 +33,7 @@ folder.
 ```ruby
 GrapeSlate.configure do |config|
   config.output_dir = 'api-docs/source'
+  config.base_path = 'http:://www.radiopaedia.org'
```

Comment:
We may want to keep this Gem public later on so for the example it may be better to use a dummy site such as `http://www.example.com`